extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/HttpClient.java

```diff
@@ -31,6 +37,46 @@
 @NonNullByDefault
 public class HttpClient {
     private int timeout = 1000;
+    private final Logger logger;
+    private final LinkedList<AsyncPutParameters> commandsQueue;
```

Comment:
Use a `LinkedBlockingQueue` and remove the synchronized statements...

Reply:
I will change that.